crates/wash-cli/src/cmd/dev/mod.rs

```diff
@@ -230,6 +230,8 @@ pub async fn handle_command(
                         .is_ok_and(|p| {
                             // Ignore Rust target directories
                             p.starts_with("target")
+                                // Ignore Go interface directories
+                                || p.starts_with("gen")
```

Comment:
Hmnnn this seems a little risky since people could have folders laid out however but let's cross that bridge when we get there.

Reply:
What about `bindings.wadge` files?

Reply:
I suppose we could ignore the bindings.wadge file too, it's just hard to do that for a specific file since that could change at any time 🤔